app/assets/javascripts/application.js

```diff
@@ -158,6 +158,68 @@ $(document).on('hidden.bs.modal', function(e) {
     OPEN_MODAL = '';
 });
 
+/**
+ * Logs properties about rendered Study Overview plot to Mixpanel via Bard.
+ *
+ * See overview diagram at:
+ * https://app.lucidchart.com/invitations/accept/3b086049-89bc-4db2-96c3-815ccc438bd5
+ *
+ * @param {String} plotType Type of plot, e.g. scatter, violin, box, dot
+ */
+function logPlot(plotType) {
+
+  var endTime = performance.now();
+  var startTimeFrontend = window.SCP.perfTimeStartPlotRender;
+
+  var startTime;
+  if (typeof window.SCP.perfTimeStartPlotTrigger === 'undefined') {
+    // Plot trigger is page navigation, e.g. for "Clusters" scatter plot
+    // startTime = window.performance.timing.navigationStart;
+    //
+    // In other words, for plots that are triggered by merely loading the page,
+    // we consider the "start time" to be the "navigation start" -- e.g.
+    // when they clicked the link to get to this page.  Clicking the link
+    // (or otherwise starting navigation) from that previous page was the
+    // *user action* that triggered this plot.  It most accurately reflects
+    // how long the user waits between doing something and seeing something
+    // they can meaningfully interact with.
+    const perfData = window.performance.timing;
```

Comment:
Yup, this one is deprecated and that one is [not supported in IE or Safari](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming#Browser_compatibility).

I see [no IE users, but 9% Safari users](https://mixpanel.com/report/2120588/view/19059/insights#report/9955232/browsers-for-home-page-scp) for home page views, per new logging by Bard client.  We [don't support Safari](https://github.com/broadinstitute/single_cell_portal_core/blob/45da8ce2f5b8f8d171ff6471aecdd0ad3adc3529/app/assets/javascripts/application.js#L430) (due to plot rendering issues, it seems), but the new API would make it costlier to support if we wanted.  I can't find any intent to deprecate `performance.timing` from Chrome (and they usually show `console.warn`s well before removing APIs), so I suspect this will work for at least several more years.

If you prefer the other API, let me know.